iso/packages.x86_64

```diff
@@ -215,6 +219,10 @@ mutter
 nautilus
 xdg-user-dirs-gtk
 
+#application to theme/configure qt apps on gtk based desktops
```

Comment:
we do not want these, no modifcations please. please stick to pure GTK

Reply:
these two applications are to provide consistency in the gtk system.
when a user installs a qt application the the theme across the system and that application won't match therefore causing non uniformity.
 kindly refer to [https://wiki.archlinux.org/title/Uniform_look_for_Qt_and_GTK_applications](this) for a better view. :)